setup.py

```diff
@@ -97,7 +97,7 @@
         "ijson>=3.1.4,<4",
         "importlib-metadata",
         "ipython>=8.5.0,<9",
-        "packaging>=21,<22",
+        "packaging>=23.0,<24",
```

Comment:
What do we even use this for? Is it pinned because of something else?

Reply:
let me know - i can create a ticket. else will just leave it for now.

Reply:
just leave it for now